content/docs/2.6/scalers/datadog.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Datadog"
+layout = "scaler"
+availability = "v2.6+"
+maintainer = "Community"
```

Comment:
Is Datadog happy to be listed as maintainer?
```suggestion
maintainer = "Datadog"
```

Basically means we'll ask Datadog to help maintain it